docs/user-guide/deploying-application/rollback-deployment.md

```diff
@@ -0,0 +1,34 @@
+# Rollback Deployment
+
+
+Deployments can be rolled back manually. After a deployment is completed, you can manually rollback a stage to a previously deployed image by retaining the same configuration or changing the configuration.
+
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/rollback-deployment/rollback-icon.jpg)
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/rollback-deployment/select-config-to-deploy.png)
+
+1. Select `Rollback` in your deployed pipeline.
+
+2. On the `Rollback` page, select a configuration to deploy from the list:
+
+| Configurations | Description |
+| --- | --- |
+| **Last saved config** | Deploy the image with the latest saved configuration. |
```

Comment:
shouldn't it be latest saved config

Reply:
In UI, it is displaying as "Last saved config", which describes the latest saved configuration. I will have a word with Mukesh once whether this can be changed as "Latest saved config"